### PR TITLE
changes to `Icon` stories, `Select`, & `Modal`

### PR DESCRIPTION
**Pull request by ryantanrk**

### CODE REVIEW

File: components/select/src/Select.svelte [hunk 1]
```diff
@@ -57,6 +57,10 @@
     outline: none;
     box-sizing: border-box;
     appearance: none;
+    background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' class='icon' viewBox='0 0 24 24' stroke-width='1.5' stroke-linecap='round' stroke-linejoin='round' %3E%3Cpath d='M6 9l6 6 6-6' /%3E%3C/svg%3E")
+      no-repeat;
+    background-size: 16px 16px;
+    background-position: 98% 50%;
```
Comment (si3nloong) [inline components/select/src/Select.svelte:63]: Should us svg in html form

**Review by si3nloong — CHANGES_REQUESTED** [review]
Correction required

File: components/modal/src/Modal.svelte [hunk 2]
```diff
@@ -14,6 +14,7 @@
   export let closable = true;
   export let hasHeader = true;
   export let hasFooter = true;
+  export let hasOverlay = true;
```
[the remark this answers is not in the document]
Comment (ryantanrk) [reply components/modal/src/Modal.svelte:17]: fixed

File: components/select/src/Select.svelte (hunk 1 above)
Comment (ryantanrk) [reply components/select/src/Select.svelte:63]: can clarify more? is it html form inside the background?

**Comment by si3nloong** [comment]
GTG

**Pull request merged**